includes/discovery/sensors/charge/dell-powervault.inc.php

```diff
@@ -8,20 +8,40 @@
 // The list is also a mix of voltages, temperatures, and state, and uses both F and C for temperatures
 // The order is not stable between software versions
 
-if (is_array($pre_cache['dell-powervault'])) {
-    foreach ($pre_cache['dell-powervault'] as $index => $entry) {
+$oids = SnmpQuery::cache()->options('-OQUbs')->walk('FCMGMT-MIB::connUnitSensorMessage')->valuesByIndex();
```

Comment:
Don't use options.  All options are available via methods.  the options method is only for backwards compatibility with yaml etc.

Reply:
done